README.adoc

```diff
@@ -1,55 +1,51 @@
 
 [cols="1,1,1",options="autowidth,header"]
 |===
 |Current Chart Version |Original Repository |Default Values
-|*{chart-version}* |{original-repo-url}/subpath/to/the/chart[Chart] |{original-repo-url}//subpath/to/the/values.yaml[`values.yaml`] (_here you can also use artifacthub.io_)
+|*{chart-version}* |{original-repo-url}[Chart] |{original-repo-url}/values.yaml[`values.yaml`]
 |===
 
 == Usage
 
-_Here you will show examples of how to call this module on your code. Note that the `depends_on` needs to be adapted to your use case._
-
-_You can start by giving a simple example and then show some others with increasing complexity and/or for specific use cases._
-
 [source,terraform]
 ----
-module "template" {
-  source = "git::https://github.com/camptocamp/devops-stack-module-template.git?ref=<RELEASE>"
+module "vault" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-vault.git?ref=<RELEASE>"
+
+  cluster_name   = local.cluster_name
+  cluster_issuer = local.cluster_issuer
+  base_domain    = local.base_domain
+  namespace      = local.vault_namespace
 
-  cluster_name            = local.cluster_name
-  argocd_namespace        = local.argocd_namespace
+  dev_root_token = random_password.vault_dev_root_token.result
 
-  # TODO Consider replacing this depends_on by our dependency_ids
-  depends_on = [
-    module.argocd_bootstrap,
-  ]
+  dependency_ids = {
+    traefik      = module.traefik.id
+    cert-manager = module.cert-manager.id
+  }
 }
-----
 
-=== Something Explained Better
+----
 
-_If there is some part of the module's configuration that needs more specific explanations, such as a very particular output on which every other module depends on, maybe consider creating a subsection to improve readability._
+=== Note
 
-NOTE: _The AsciiDoc syntax provides a way to write admonitions such as this one, so use it if you think it is useful. The https://docs.antora.org/antora/latest/asciidoc/admonitions/[official documentation] for Antora Docs provides more examples._
+This module is experimental as is intended for use as part of a KinD setup. It should never be used in production as Vault is only supported in dev mode for now.
 
 == Technical Reference
 
-_This section will contain the more technical details of the module. It is mainly generated automatically by Terraform Docs (everything that is between the comments `BEGIN_TF_DOCS` / `END_TF_DOCS`). However, it is recommended to, at the bare minimum, add a dependencies section to explain the reason on why this module could depend on another one from the DevOps Stack._
-
-_You can safely delete the content between the comments `BEGIN_TF_DOCS` / `END_TF_DOCS` and `BEGIN_TF_TABLES` / `END_TF_TABLES` when creating your on module._
-
 === Dependencies
 
-==== `module.argocd_bootstrap`
+==== `module.traefik`
+==== `module.cert-manager`
 
 This module must be one of the first ones to be deployed and consequently it needs to be deployed after the module `argocd_bootstrap`.
```

Comment:
```suggestion
```

This sentence has no place here since you removed the dependency of the bootstrap Argo CD.